### ISSUE TITLE
split out data package

### ISSUE THREAD

**Issue by mdsumner** [mdsumner/ozplot#10]
I'd hoped to not have to do this, but CRAN wants packages under 5Mb *unless* it's a data package, and hence won't change very often and clog up the disks with archives. 

* rejig `ozmaps.data` to be these layers, it means that package should really use piggyback and tighten that up a lot, keep the original stuff and ship the simplified versions
* `ozmaps` will be bare minimum, whatever that means - states, commonwealth electoral - probably, maybe LGA - all other layers kept external

(that means a bit of back tracking given how I first started this process)

**Comment by hadley** [comment]
Have you tried any generalisation to reduce the file size? The level of detail in the coastlines seems pretty high. Maybe this package could provide fairly low-res approximations, and you could provide a separate hi-res data package?

**Comment by mdsumner** [comment]
Yes these are simplified from the sources in [ozmaps.data]( https://github.com/mdsumner/ozmaps.data ) - it's kind of halfway baked atm, I think ABS has updated the formats recently so all needs another look.
